test/bug_patches/chi2inv.m

```diff
@@ -0,0 +1,71 @@
+## Copyright (C) 2012 Rik Wehbring
```

Comment:
@TomMaullin: is it normal that you have included this chi2inv.m file in the version-control?

Reply:
Ah apologies... I thought I had removed this! I'll remove it now!